systest/incremental-restore/incremental_restore_test.go

```diff
@@ -45,16 +47,17 @@ func TestIncrementalRestore(t *testing.T) {
 
 	hc, err := c.HTTPClient()
 	require.NoError(t, err)
-	require.NoError(t, hc.LoginIntoNamespace(dgraphtest.DefaultUser, dgraphtest.DefaultPassword, 0))
+	require.NoError(t, hc.LoginIntoNamespace(dgraphtest.DefaultUser,
+		dgraphtest.DefaultPassword, x.GalaxyNamespace))
 
 	uids := []int{1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20}
 	c.AssignUids(gc.Dgraph, uint64(len(uids)))
 	require.NoError(t, gc.SetupSchema(`money: [int] @index(int) @count .`))
 
 	for i := 1; i <= len(uids); i++ {
 		for j := 1; j <= i; j++ {
-			rdfs := fmt.Sprintf(`<%v> <money> "%v" .`, j, i)
-			_, err := gc.Mutate(rdfs)
+			mu := &api.Mutation{SetNquads: []byte(fmt.Sprintf(`<%v> <money> "%v" .`, j, i)), CommitNow: true}
```

Comment:
Why?

Reply:
Through this API, we can do both deletion and set in a mutation.